components/Layout/HomeHeader/HomeHeaderItems.vue

```diff
@@ -124,12 +87,15 @@
 <script lang="ts">
 import Vue from 'vue';
 import Swiper, { SwiperOptions } from 'swiper';
+import { VImg } from 'vuetify/lib';
 import { mapActions } from 'vuex';
 import { BaseItemDto, ImageType } from '@jellyfin/client-axios';
 import htmlHelper from '~/mixins/htmlHelper';
 import imageHelper from '~/mixins/imageHelper';
 
 export default Vue.extend({
+  // eslint-disable-next-line vue/no-unused-components -- Used in child functional components
+  components: { VImg },
```

Comment:
Can't we import those in the child components?

Reply:
Functional components cannot use `components`. We import it in the parent, so that children can use them (Otherwise, the component is undefined).